free-1.md

```diff
@@ -57,12 +73,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+The developer's attentions would probably be distracted by creating flavor pairings and figuring out a method of efficient searching through a connected graph. It would get particularly confusing to introduce categories (potentially in the form of attribute tags) for each different flavor or food. It would be easy to spend too much time focusing on this setup instead of the actual language for the user.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+As discussed in the previous question, this is potentially a huge project. It would be very ambitious to create a flavor pairing graph, especially if users were allowed to define their own pairings. It has the potential to be an incredible service, but it would be difficult to implement much functionality over such a complex data set in the timeframe of this project.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This would be a good idea for a project because it could be a great DSL to present users with a lot of food pairings tailored to their specific needs. However, as previously discussed, the scope of the project could end up being very significant and the focus might shift off of the DSL itself.
+
```

Comment:
I really like the idea, but it definitely doesn't seem realistic (if you wanted to complete it entirely).